spec/models/manageiq/providers/openshift/container_manager/refresher_spec.rb

```diff
@@ -142,6 +142,22 @@
     end
   end
 
+  it 'will skip container_images if get_container_images = false' do
+    stub_settings(Settings.to_hash.deep_merge(
```

Comment:
There is no `stub_settings_merge` in `euwe` so had to do this manually (this is what `stub_settings_merge` does internally anyway)